src/controllers/update/update.manage.trusts.tell.us.about.the.individual.controller.ts

```diff
@@ -29,12 +29,18 @@ const getPageProperties = (trust, formData, trustee: TrustIndividual, errors?: F
       title: 'Tell us about the individual',
     },
     pageData: {
-      trustData: { trustName: trust?.trust_name },
+      trustData: {
+        trustName: trust?.trust_name,
+        creationDateDay: trust?.creation_date_day,
```

Comment:
Wondering why 'creationDate' features in the names here and not 'ceasedDate'?

Reply:
one of the validation rules is to check that the cease date is after the creation date. As there might be more than one trust i figured it would be easier to get the creation date at this point and pass it into the html form where it can be used in validation of the ceased date.

Reply:
Is this generally the right approach - anything that we need to validate later on (when form is POSTed) has to be present in the form HTML?

I don't think we do that normally, rather, the custom validator gets hold of the app data and picks out the data it needs to validate against.

Reply:
i'll look into using this approach.

Reply:
a test comment due to problems with comments (permissions)

Reply:
changed the validation to lookup the trust creation date